chainer/functions/normalization/batch_normalization.py

```diff
@@ -15,6 +17,227 @@
 import chainerx
 
 
+class _BatchNormalizationBackend(six.with_metaclass(ABCMeta)):
```

Comment:
As a convention, Chainer does not use `abc` for abstract classes, but just inherit from `object`.
As we have dropped Python 2, inheriting from `object` is not necessary either.
How about simply `class _BatchNormalizationBackend:`?

Reply:
Addressed in 56f3d93